corehq/apps/export/models/new.py

```diff
@@ -920,6 +933,53 @@ def get_filters(self):
         return []
 
 
+class SMSExportInstance(ExportInstance):
+    type = SMS_EXPORT
+    identifier = None
+    name = "Messages"
+
+    @classmethod
+    def _new_from_schema(cls, schema):
+        columns = [
```

Comment:
Could also make these nametuple but this felt readable to me since it's only used here